### PR TITLE
Add $baddog assetlist.json

### PR DESCRIPTION
**Pull request by SmolFatality**
Adding baddog asset information. wasn't automatically pulled from cosmos registry.

### CODE REVIEW

File: osmosis/assetlist.json [hunk 1]
```diff
@@ -1359,6 +1359,47 @@
...
+          "aliases": ["uwoof"]
+        },
+        {
+          "denom": "achihuahuawifhat",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/2FFE07C4B4EFC0DDA099A16C6AF3C9CCA653CC56077E87217A585D48794B0BC7",
+      "name": "BADDOG",
```
Comment (JeremyParish69) [inline osmosis/assetlist.json:1378]: Why is the name different?

Comment (SmolFatality) [reply osmosis/assetlist.json:1378]: Sorry fixing

**Review by SmolFatality — COMMENTED** [review]
I fixed the name, thanks for reviewing Jeremy!

File: osmosis/assetlist.json (hunk 1 above)
Comment (SmolFatality) [reply osmosis/assetlist.json]: fixed!

**Comment by JeremyParish69** [comment]
Oops, had to revert @SmolFatality . Many details are incorrect--the base denom and the ibc data are not right.

**Comment by SmolFatality** [comment]
Okay I will try to get it fixed correctly before I try to pr again. Sorry.

**Pull request merged**